### DISCUSSION TITLE
Welcome to ClimaOcean.jl Discussions!

### DISCUSSION THREAD

**Discussion by navidcy** [CliMA/ClimaOcean.jl#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cmichelenstrofer** [comment]
Hi! I watched your JuliaCon Workshop. Oceananigans is amazing! I love the implementation. I work on marine renewable energy. I am new to Julia, and am looking at writing a boundary element method to solve the diffraction and radiation equations in Julia. The goal would be to (1) tackle the two-language problem and (2) to have a differentiable framework for modeling wave energy converters. I am planning at looking in detail at Oceananigans and other projects to see how much I can use. Hopefully the re-use power of Julia is as good as they say and I can find existing solutions for many of the steps required in modeling wave energy converters. And while I don't work on the resource characterization side of marine energy, I am wondering if Oceananigans could be used instead of some of the codes we currently use for wave resource characterization like NOAA's Wave Watch III or SWAN. Thanks again for making this amazing package public and maintaining it! :)

**Reply by glwagner** [reply]
Hi @cmichelenstrofer ! Oceananigans (or the Oceananigans setups provided by ClimaOcean) will not be a replacement for Wave Wave III --- ClimaOcean is a physical-space, 3D model for ocean currents and tracers, while Wave Watch III is a 2D, quasi-spectral model just for ocean surface waves.

But keep us updated on the progress of your project and if there might be any other ways you can use Oceananigans or ClimaOcean!
